website/content/docs/release-notes/1.12.0.mdx

```diff
@@ -0,0 +1,136 @@
+---
+layout: docs
+page_title: 1.12.0
+description: |-
+  This page contains release notes for Vault 1.12.0
+---
+
+# Vault 1.12.0 Release Notes
+
+**Software Release date:** Oct. 12, 2022
+
+**Summary:** Vault Release 1.12.0 offers features and enhancements that improve the user experience while solving critical  issues previously encountered by our customers. We are providing an overview  of improvements in this set of  release notes.
+
+We encourage you to upgrade to the latest release of Vault to take advantage of the new benefits provided. With this latest release, we offer solutions to critical feature gaps that were identified previously. Please refer to the [Changelog](https://github.com/hashicorp/vault/blob/main/CHANGELOG.md) within the Vault release for further information on product improvements, including a comprehensive list of bug fixes.
+
+Some of these enhancements and changes in this release include the following:
+
+
+- Vault Enterprise now supports PKCS#11 provider plugin (client library) functionality.
+- Vault Enterprise can now manage keys for Oracle TDE. This requires the Advanced Data Protection license.
+- PKI Key revocation improvements are made to Vault’s PKI engine, introducing a new OCSP responder and automatic CRL rebuilding (with up-to-date Delta CRL), that offers significant performance and data transfer improvements to revocation workflows.
+- BYOK in Transform engines now allow users to import their keys generated elsewhere.
+- KMIP Server Profile adds support for additional operations, allowing  Vault to claim support for the baseline server profile.
+- Transform secrets engine supports time-based auto-key rotation for tokenization.
+- Path and Role-based Quotas extend the existing Vault Quota support by allowing quotas to be extended to the API path suffixes and auth mount roles.
+- Licensing termination behavior has changed where non-evaluation licenses (production licenses) will no longer have a termination date.
+- TFVP includes improvements made to the Vault Version Detection and Auth engine.
```

Comment:
Version detection is a new feature and should be "imprevments to Auth Engines".

Reply:
Re-wrote the whole sentence ([commit](https://github.com/hashicorp/vault/pull/17237/commits/4583f44c3151389f3f470d6abe502b37739e04be))